lib/platform/github/index.ts

```diff
@@ -97,6 +98,19 @@ export async function initPlatform({
   } else {
     logger.debug('Using default github endpoint: ' + defaults.endpoint);
   }
+
+  config.isGhe = URL.parse(defaults.endpoint).host !== 'api.github.com';
+  if (config.isGhe) {
+    const gheHeaderKey = 'x-github-enterprise-version';
+    const gheQueryRes = await githubApi.head('/');
```

Comment:
hopefully this won't throw on GHE. Should we set `throwHttpErrors=false`? So we don't throw here?